.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - 12.13.0
+
+services:
+  - postgresql
+
+before_script:
+  - psql -c 'create database spinners;' -U postgres
+
+cashe: npm
+
+install:
+  - npm install
+
+script:
+  - npm test  
+
+after-script:
+  - npm run migrate
```

Comment:
Migration should run before running the test and not after.

Reply:
Should I put inside before_script? just before creating a database?